lib/components/Input/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./base";
```

Comment:
i'm guessing that this file should export both `base` and `text-field`. :)

Also, i thought there should be a `base.tsx` file in the main directory (like in `Typography/`), which would contain the main `Input` component.